src/overlays/actors/ovl_Dm_Char02/z_dm_char02.c

```diff
@@ -130,15 +145,16 @@ void DmChar02_Update(Actor* thisx, PlayState* play) {
     DmChar02* this = THIS;
 
     SkelAnime_Update(&this->skelAnime);
-    this->unk2F0 = this->unk2F0;
+    this->unk_2F0 = this->unk_2F0;
```

Comment:
This is required to match? Wouldn't IDO recognize this doesn't do anything and omit it anyway?

Reply:
It is required to match, and is also the only usage of the field. Very weird.

Reply:
Yeah this in a few actors like this. I have zero idea why it's necessary, but it is. Thanks IDO!